appveyor.yml

```diff
@@ -0,0 +1,17 @@
+environment:
+  PYTHON: C:\\Python37-x64\\python.exe
+  PYTHONPATH: C:\\Python37-x64\\Scripts
+  SQFLINT: C:\\Python37-x64\\Scripts\\sqflint.exe
+
+build_script:
+  - "%PYTHON% build.py -p -z build"
```

Comment:
Nitpick: you just need `-z` as it impiles `-p`.

Reply:
Removed `-p`